packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.types.ts

```diff
@@ -94,6 +115,34 @@ export interface IScrollablePaneStyles {
   contentContainer: IStyle;
 }
 
+/**
+ * {@docCategory ScrollablePane}
+ */
+export const StickyContainerBehaviorType = {
+  /**
+   * This is the default behavior.
+   */
+  Default: 'Default' as 'Default',
+
+  /**
+   * Sticky component(s) will become sticky or non-sticky based on scrolling.
+   * The calculation which determine if a Sticky component is sticky or non-sticky,
+   * are done after user interaction (scrolling) and don't affect page load time.
+   */
+  StickyOnScroll: 'StickyOnScroll' as 'StickyOnScroll',
+
+  /**
+   * Sticky component(s) will always be sticky independent of scrolling.
+   * There are no calculations done as the component(s) would always be sticky.
+   */
+  StickyAlways: 'StickyAlways' as 'StickyAlways'
+};
+
+/**
+ * {@docCategory ScrollablePane}
+ */
+export type StickyContainerBehaviorType = typeof StickyContainerBehaviorType[keyof typeof StickyContainerBehaviorType];
```

Comment:
It seems that you're exporting 2 things of the same name.

I think this type should be:

```tsx
export type IStickyContainerBehaviorType = 'default' | 'onScroll' | 'always';
```

That way, you don't need to import anything and you can just pass in a string.